pyproject.toml

```diff
@@ -34,6 +42,12 @@ line-length = 127
 target-version = ['py312']
 include = '\.pyi?$'
 
+[tool.pytest.ini_options]
+DJANGO_SETTINGS_MODULE = "pems.settings"
+markers = [
+    "request_path: use with session_request to initialize with the given path",
```

Comment:
Do we need this?

Reply:
Thanks for catching this, we actually don't need `markers` so I removed it.